ts/main.ts

```diff
@@ -1,14 +1,23 @@
 document.addEventListener('DOMContentLoaded', () => {
   const $form = document.getElementById('entry-form') as HTMLFormElement;
   const $titleInput = document.getElementById('title') as HTMLInputElement;
-
   const $photoUrlInput = document.getElementById(
     'photo-url',
```

Comment:
I may have missed this earlier, but your `DOMContentLoaded` event listener should not be over all of your `ts` file. Your `DOMContentLoaded` event listener should just have the part where you are appending entries to the DOM, swapping your view, and then toggling no entries.